### PR TITLE
Add command for building pipeline for OpenRISC projects

### LINKED ISSUE #66: Move shared code of OpenRISC Pipelines to LibreCores Pipeline Library
OpenRISC projects i.e mor1kx and or1k-marocchino CI involves multi-branch-pipeline structure which also uses librecores-ci and a [librecores-ci-openrisc](https://github.com/librecores/docker-images/tree/master/librecores-ci-openrisc) docker image .
This issue involves Detaching reusable logic of [mor1kx](https://github.com/openrisc/mor1kx/blob/master/Jenkinsfile)  and [or1k-marochhino](https://github.com/openrisc/or1k_marocchino/blob/master/Jenkinsfile) to librecores-pipeline-lib. This will simplify the pipeline libraries

### PR DESCRIPTION
**Pull request by Nancy-Chauhan**
Fixes https://github.com/librecores/librecores-ci-jenkins-server/issues/66

- [x] Move shared code of OpenRISC Pipelines to LibreCores Pipeline Library
- [x] Create DSL for configuring CI jobs

### CODE REVIEW

**Comment by Nancy-Chauhan** [comment]
@oleg-nenashev @stffrdhrn Hello , This works for moving shared code for openrisc lib for Jenkinsfile : https://github.com/Nancy-Chauhan/mor1kx/blob/openrisc-lib/Jenkinsfile . I have tested on my local instance

File: vars/visualisingTest.groovy [hunk 1]
```diff
@@ -0,0 +1,18 @@
+#!/usr/bin/env groovy
```
Comment (oleg-nenashev) [inline vars/visualisingTest.groovy]: Still not clear why this wrapper is needed ATM

**Comment by Nancy-Chauhan** [comment]
@oleg-nenashev 

I tried building a DSL for defining OpenRISC pipelines, please let me know if this looks ok

```groovy
@Library('lcci') _

openrisc_ci {
        job('verilator') {
            job 'verilator' 
        }
        
        job('testing-1') {
            job 'or1k-tests'
            sim 'icarus'
            pipeline 'CAPPUCCINO'
            expectedFailures 'or1k-cy'
        }

		// ...
}
```

Will update README accordingly

File: vars/visualisingTest.groovy (hunk 1 above)
Comment (Nancy-Chauhan) [reply vars/visualisingTest.groovy]: Removed it as it is not used

**Review by oleg-nenashev — APPROVED** [review]
I think this is a good start, and we can keep moving incrementally

**Pull request merged**